external-builds/pytorch/pytorch_torch_repo.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-"""Checks out and builds PyTorch against a built from source ROCM SDK.
+"""Checks out PyTorch.
```

Comment:
Can you also make these doc updates to these other files?

* https://github.com/ROCm/TheRock/blob/main/external-builds/pytorch/pytorch_audio_repo.py
* https://github.com/ROCm/TheRock/blob/main/external-builds/pytorch/pytorch_vision_repo.py

Reply:
Done.